cmd/kcoin/genesis_options.go

```diff
@@ -0,0 +1,128 @@
+package main
```

Comment:
maybe `knode/genesis/networks.go`?

not cmd, command should just dispatch functionality